### DISCUSSION TITLE
0011 XLS-11d: Retiring Amendments

### DISCUSSION THREAD

**Discussion by WietseWind** [XRPLF/XRPL-Standards#19] [XLS Proposals]
## The earlier version can be found by looking at the edit history.

**Comment by WietseWind** [comment]
@mDuo13 This was a PR, moved it here as we're working with issues for discussion and WIP until definitive. We decided to roll with Issues until everything is definitive. We'll then commit the final standard to the repository.

**Comment by mvadari** [comment]
@mDuo13 @WietseWind should this spec be moved into PR and kept in the repo or should it be closed? Note that this does not mean that the spec has been accepted/implemented/finalized.

I will move it to PR by copy-pasting the existing draft if there is no response in the next week.

**Reply by WietseWind** [reply]
I still believe retiring amendments is a good idea.

**Reply by mvadari** [reply]
Pretty sure this was done for the listed amendments.
